### PR TITLE
WEB-9710: 3 up cards styling

### PR DESCRIPTION
**Pull request by zoeyleqa**

### CODE REVIEW

File: scss/partials/_cards.scss [hunk 1]
```diff
@@ -63,13 +69,13 @@
     max-width: 100%;
   }
 
-  &__subtitle {
+  &__lower &__subtitle {
```
Comment (fienen) [inline scss/partials/_cards.scss:72]: This selector feels not BEM, since it results in two chained elements.

**Review by fienen — CHANGES_REQUESTED** [review]

File: scss/partials/_cards.scss (hunk 1 above)
Comment (zoeyleqa) [reply scss/partials/_cards.scss:72]: I was trying to add more specificity score to the element to overwrite the default `<p>` tag color. Should I think of another work around @fienen?

Comment (fienen) [reply scss/partials/_cards.scss:72]: I'd just toss on a new modifier instead. `--somecolor` or something like that.

Comment (nnedevn) [reply scss/partials/_cards.scss:72]: For what it's worth - this is another way to increase specificity. 
https://css-tricks.com/the-sass-ampersand/#doubling-up-specificity

Comment (zoeyleqa) [reply scss/partials/_cards.scss]: think I fixed it right ... Could you guys double check for me please?

**Pull request merged**